Python/importdl.c

```diff
@@ -413,6 +413,8 @@ _PyImport_GetModuleExportHooks(
     if (exportfunc) {
         *modexport = (PyModExportFunction)exportfunc;
         return 2;
+    } else if (PyErr_Occurred()) {
+        PyErr_Clear();
     }
```

Comment:
No need to check for PyErr_Occurred():

```suggestion
    }
    PyErr_Clear();
```